### ISSUE TITLE
Customizing transition animation on tab navigation

### ISSUE THREAD

**Issue by panda0603** [react-navigation/react-navigation#6828]
I've already searched for issues regarding transition animation for bottom tab navigation. 

Currently, there seems to be no way to make any transition during navigating to different tabs, except for using materialTopTabNavigator(though it is not customizable, which is what I need). 

I would really like to make a transition(just like stack navigator) on tab navigation. I have already tried setting transition for stack navigator that is inside tab navigator, but doesn't work that way. 

Is there any possible way to customize transition animation for tab navigation at the moment?
If building custom navigator is the only way, it would be nice to provide any examples that implements animation. At the moment doc is hard to follow on building custom navigations.

| software                      | version |
| ----------------------------- | ------- |
| iOS or Android                | Both
| @react-navigation/native      | 5.0.2
| @react-navigation/bottom-tabs | 5.0.2
| react-native-screens          | 2.0.0
| react-native                  | 0.61.5

**Comment by WoLewicki** [comment]
It looks like it is not possible in bottom-tabs for now. You can add it to React-Navigation canny as a feature request if you want: https://react-navigation.canny.io/feature-requests.
